Vagrantfile

```diff
@@ -20,7 +22,9 @@ Vagrant.configure(2) do |config|
     puppet.manifests_path = "puppet"
     puppet.module_path = "puppet/modules"
     puppet.facter = {
-    	"quickstart_domain" => config.vm.hostname
+        "hostname"          => config.vm.hostname,
```

Comment:
Hostname should be a built-in fact. https://docs.puppetlabs.com/facter/latest/core_facts.html#hostname